apps/api/src/domain/models/photograph/repositories/arango-photograph-query-repository.integration.spec.ts

```diff
@@ -89,12 +97,13 @@ const buildPhotographEventHistory = (
     });
 };
 
-const photographViews = photographIds.map((id) => {
+const photographViews = photographIds.map((id, index) => {
```

Comment:
I need to take a look at the test setup and make sure it's leveraging the latest pattern.